devdoc/twodarray_requirements.md

```diff
@@ -0,0 +1,167 @@
+# `two_d_array` requirements
+============================
+
+## Overview
+
+`TWODARRAY` is a module provides a 2D array with variable size of rows and columns. Each row has the same size to ensure easier access. The first dimension of array is resizable, it contains the pointer to the second dimension of array. The pointer is initialized to NULL and points to the allocated memory address when there's not enough space. When the memory stored in a row is not needed any more, the entire row shall be freed and set back to `NULL`.
```

Comment:
>TWODARRAY [](http://example.com/codeflow?start=1&length=9)

You should have this as TWO_D_ARRAY or TWO_DARRAY, just so it's easier to read.

Reply:
replaced with TWO_D_ARRAY